tests/openassetio/test/manager/resources/plugins/StubManager.py

```diff
@@ -54,9 +54,9 @@ def initialize(self, hostSession):
         # pylint: disable=unused-argument
         pass
 
-    def managementPolicy(self, specifications, context, hostSession):
+    def managementPolicy(self, traitSets, context, hostSession):
         # pylint: disable=unused-argument
-        return [constants.kIgnored for _ in range(len(specifications))]
+        return [constants.kIgnored for _ in range(len(traitSets))]
```

Comment:
[consider][minor] Not caused by this PR but this is rather verbose
```suggestion
        return [constants.kIgnored for _ in traitSets]
```
or
```suggestion
        return [constants.kIgnored] * len(traitSets)
```